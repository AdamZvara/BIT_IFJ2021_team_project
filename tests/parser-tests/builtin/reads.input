require "ifj21"


function main()
	local s : string = ""
	write("ZADEJTE VELKE PISMENO: ")
	s = reads()
	while s ~= "X" do
		write(s," is wrong input\n")
		s = reads()
	end
	write(s)
end


main()
