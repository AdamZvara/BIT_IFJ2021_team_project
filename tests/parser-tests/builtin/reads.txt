A
B
C
D
E
F
X
E
Q
E
