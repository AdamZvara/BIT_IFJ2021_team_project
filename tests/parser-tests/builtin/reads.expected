ZADEJTE VELKE PISMENO: A is wrong input
B is wrong input
C is wrong input
D is wrong input
E is wrong input
F is wrong input
X